features/command_line/order.md

```diff
@@ -13,6 +13,8 @@ order of groups at each level is randomized.
 
 With `rand` you can also specify a seed.
 
+The option `modification_time` will run the most recent modified files first. You can combine it with `--next-failure` to find the most recent failing spec.
```

Comment:
This line needs to be wrapped as above, and you need to mention it on line 2, I'd also suggest:
```suggestion
Use `modification_time` to run the most recently modified files first. You can combine it with `--next-failure` to find the most recent failing spec.
```